src/Bluejay.asm

```diff
@@ -446,670 +444,662 @@ $include (Modules\Settings.asm)
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 
-
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 ;
 ; Main program entry point
 ;
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 pgm_start:
-	mov	Flash_Key_1, #0			; Initialize flash keys to invalid values
-	mov	Flash_Key_2, #0
-	mov	WDTCN, #0DEh				; Disable watchdog (WDT)
-	mov	WDTCN, #0ADh
-	mov	SP, #Stack				; Initialize stack (16 bytes of indirect RAM)
-	IF MCU_TYPE < 2
-		orl	VDM0CN, #080h			; Enable the VDD monitor
-	ENDIF
-	mov	RSTSRC, #06h				; Set missing clock and VDD monitor as a reset source if not 1S capable
-	mov	CLKSEL, #00h				; Set clock divider to 1 (Oscillator 0 at 24MHz)
-	call	switch_power_off
-	; Ports initialization
-	mov	P0, #P0_INIT
-	mov	P0MDIN, #P0_DIGITAL
-	mov	P0MDOUT, #P0_PUSHPULL
-	mov	P0, #P0_INIT
-	mov	P0SKIP, #P0_SKIP
-	mov	P1, #P1_INIT
-	mov	P1MDIN, #P1_DIGITAL
-	mov	P1MDOUT, #P1_PUSHPULL
-	mov	P1, #P1_INIT
-	mov	P1SKIP, #P1_SKIP
-	mov	P2MDOUT, #P2_PUSHPULL
-IF MCU_TYPE >= 1
-	; Not available on BB1
-	mov	SFRPAGE, #20h
-	mov	P2MDIN, #P2_DIGITAL
-	IF MCU_TYPE == 1
-		; Not available on BB51
-		mov	P2SKIP, #P2_SKIP
-	ENDIF
-	mov	SFRPAGE, #00h
-ENDIF
-	Initialize_Crossbar				; Initialize the crossbar and related functionality
-	call	switch_power_off			; Switch power off again, after initializing ports
-
-	; Clear RAM
-	clr	A						; Clear accumulator
-	mov	Temp1, A					; Clear Temp1
-	clear_ram:
-	mov	@Temp1, A					; Clear RAM address
-	djnz	Temp1, clear_ram			; Decrement address and repeat
-
-	call	set_default_parameters		; Set default programmed parameters
-	call	read_all_eeprom_parameters	; Read all programmed parameters
-	call	decode_settings			; Decode programmed settings
-
-	; Initializing beeps
-	clr	IE_EA					; Disable interrupts explicitly
-	call	wait100ms					; Wait a bit to avoid audible resets if not properly powered
-	call	play_beep_melody			; Play startup beep melody
-	call	led_control				; Set LEDs to programmed values
-
-	call	wait100ms					; Wait for flight controller to get ready
-
-
-;**** **** **** **** **** **** **** **** **** **** **** **** ****
-;
-; No signal entry point
-;
-;**** **** **** **** **** **** **** **** **** **** **** **** ****
+    mov                           Flash_Key_1,  #0 ; Initialize flash keys to invalid values
+    mov                           Flash_Key_2,  #0
```

Comment:
There are lots of spaces after instructions

Reply:
Yes, that's what I do, but I normalize it over the whole file (not only considering instructions though). I could make another pass and allign it to the longest instruction, so basically pad everything to 4 chars (plus space).